src/ui/Quickview/Quickview.ts

```diff
@@ -212,7 +230,21 @@ export class Quickview extends Component {
         }
         return DomUtils.getBasicLoadingAnimation();
       }
-    )
+    ),
+
+    /**
+     * Specifies where to position the tooltip compared to the element
+     * Values are `auto`, `top`, `left`, `right`, `bottom`.
+     * You can also append these variations: `-start`, `-end` which correspond to the alignment (by default they are centered)
+     *
+     * Default value is `bottom`.
+     *
+     * **Example:**
+     * `top-start` tooltip is on top of the `Quickview` button, aligned on the left (start)
+     */
+    tooltipPlacement: ComponentOptions.buildCustomOption<Placement>((value: Placement) => value, {
```

Comment:
I always like it when we are able to 100% the public API of a component.

Options definitely are something that over the years can be "forgotten" when refactoring and we suddenly realize they are not being passed down properly in the chain.

We definitely need a test for this.

Reply:
Added!